taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -0,0 +1,231 @@
+            >
+                <Grid item xs={3} sx={{mt: 4}}>
+                     <Typography variant="h4">Search Results</Typography>
+                </Grid>
+                <Typography variant="h6" sx={{mt: 2, mb: 1}}>
+                Number of nodes found: {nodes.length}
+                </Typography>
+                {/* Table for listing all nodes in taxonomy */}
+                <TableContainer sx={{width: 375}} component={Paper}>
+                    <Table>
+                        <TableHead>
+                        <TableRow>
+                            <Stack direction="row" alignItems="center">
+                                <TableCell align="left">
+                                <Typography variant="h6">
+                                    Nodes
+                                </Typography>
+                                </TableCell>
+                                <IconButton sx={{ml: 1, color: "#808080"}} onClick={handleOpenAddDialog}>
+                                    <AddBoxIcon />
+                                </IconButton>
+                            </Stack>
+                            <TableCell align="left">
+                            <Typography variant="h6">
+                                Action
+                            </Typography>
+                            </TableCell>
+                        </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {nodes.map((nodeID) => (
+                                <TableRow
+                                key={nodeID}
+                                >
+                                    <TableCell align="left" component="td" scope="row">
+                                        <Typography variant="subtitle1">
+                                            {nodeID}
+                                        </Typography>
+                                    </TableCell>
+                                    <TableCell align="left" component="td" scope="row">
+                                        <IconButton onClick={event => handleClick(event, nodeID) } aria-label="edit">
+                                            <EditIcon color="primary"/>
+                                        </IconButton>
+                                    </TableCell>
+                                </TableRow>
+                            ))}
+                        </TableBody>
+                    </Table>
+                </TableContainer>
+                {/* Dialog box for adding nodes */}
+                <Dialog open={openAddDialog} onClose={handleCloseAddDialog}>
+                    <DialogTitle>Add a node</DialogTitle>
+                    <DialogContent>
+                    <Stack direction="row" alignItems="center" sx={{mt: 1, ml: 2, mr: 2, mb: 2}}>
+                        <Typography>Type of node</Typography>
+                        <FormControl sx={{ml: 6.5}}>
```

Comment:
could we use integer?

Reply:
Yep, I tried my level best to keep everything as integers. Only one textbox should have a decimal margin, since it didn't align properly in my dialog component.